neptune/codelist.go

```diff
@@ -158,29 +158,39 @@ list exists. It returns a wrapped error if a Code is found that does not have a
 */
 func (n *NeptuneDB) GetCodes(ctx context.Context, codeListID, edition string) (*models.CodeResults, error) {
 	qry := fmt.Sprintf(query.GetCodes, codeListID, edition)
-	codeResponses, err := n.getVertices(qry)
+	values, err := n.getStringList(qry)
 	if err != nil {
 		return nil, errors.Wrapf(err, "Gremlin query failed: %q", qry)
 	}
-	if len(codeResponses) == 0 {
+	if len(values) == 0 {
 		return nil, driver.ErrNotFound
 	}
-	codeResults := &models.CodeResults{
+
+	records, err := createRecords(values, 2)
```

Comment:
`2` is a magic number and is not clear what is means in this context or why the value `2` is needed. Consider making a var or const for this.